clang/test/CIR/Lowering/data-member.cir

```diff
@@ -1,13 +1,13 @@
 // RUN: cir-opt -cir-to-llvm -o - %s | FileCheck -check-prefix=MLIR %s
-// RUN: cir-translate -cir-to-llvmir --disable-cc-lowering -o - %s  | FileCheck -check-prefix=LLVM %s
+// RUN: cir-translate -cir-to-llvmir --target x86_64-unknown-linux-gnu --disable-cc-lowering -o - %s  | FileCheck -check-prefix=LLVM %s
 
 !s32i = !cir.int<s, 32>
 !s64i = !cir.int<s, 64>
 !structT = !cir.struct<struct "Point" {!cir.int<s, 32>, !cir.int<s, 32>, !cir.int<s, 32>}>
 
 module @test attributes {
   cir.triple = "x86_64-unknown-linux-gnu",
-  llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+  dlti.dl_spec = #dlti.dl_spec<i128 = dense<128> : vector<2xi64>, f80 = dense<128> : vector<2xi64>, !llvm.ptr<271> = dense<32> : vector<4xi64>, !llvm.ptr<272> = dense<64> : vector<4xi64>, i64 = dense<64> : vector<2xi64>, f16 = dense<16> : vector<2xi64>, i32 = dense<32> : vector<2xi64>, f128 = dense<128> : vector<2xi64>, !llvm.ptr<270> = dense<32> : vector<4xi64>, f64 = dense<64> : vector<2xi64>, !llvm.ptr = dense<64> : vector<4xi64>, i1 = dense<8> : vector<2xi64>, i8 = dense<8> : vector<2xi64>, i16 = dense<16> : vector<2xi64>, "dlti.stack_alignment" = 128 : i64, "dlti.endianness" = "little">
```

Comment:
In the future, can we make `dlti.dl_spec` be automatically computed if non-exist and we are passing the triple in? It'd be nice to be able to write small unity tests without having to copy-n-paste the spec if there's a convenient way to generate it out of the triple.

Reply:
Sorry for being vague about this part: `dl_spec` in `cir-translate` code path is already auto computed based on the triple. But `cir-opt` support comes later with a different code path. So test cases referencing `cir-opt` still requires explicit `dl_spec`.

Btw, do you think it's reasonable remove `cir-opt` part in tests with a RUN line of `cir-translate` given? Since the equivalent solution for `cir-opt` might be a bit dirtier than this patch, we could use `cir-translate` as the preference for testing scenarios to avoid it.

Reply:
Thanks for the clarification!

> Btw, do you think it's reasonable to remove cir-opt part in tests, given that a RUN line of cir-translate still exists?

Sounds good